src/views/about/Team.vue

```diff
@@ -11,40 +11,35 @@
     </p>
     <Gallery>
       <Portrait
-        link="https://vincentrubinetti.com"
-        :image="vince"
-        name="Vincent Rubinetti"
-        role="Frontend Developer"
+        name="Chunlei Wu"
+        role="Principal Investigator"
+        link="https://wulab.io/the-team-chunlei-wu"
+        :image="chunlei"
       />
+
       <Portrait
-        link="https://vincentrubinetti.com"
-        :image="vince"
-        name="Vincent Rubinetti"
-        role="Frontend Developer"
+        name="Ricardo Avila"
+        role="Research Programmer"
+        link="https://wulab.io/the-team-ricardo-avila"
+        :image="ricardo"
       />
       <Portrait
-        link="https://vincentrubinetti.com"
-        :image="vince"
-        name="Vincent Rubinetti"
-        role="Frontend Developer"
+        name="Dongbo Hu"
+        role="Backend Developer"
+        link="https://github.com/dongbohu"
+        :image="dongbo"
       />
       <Portrait
-        link="https://vincentrubinetti.com"
-        :image="vince"
-        name="Vincent Rubinetti"
-        role="Frontend Developer"
+        name="Xinghua (Jerry) Zhou"
+        role="Research Programmer"
+        link="https://wulab.io/the-team-jerry-zhou"
+        :image="jerry"
       />
       <Portrait
-        link="https://vincentrubinetti.com"
-        :image="vince"
         name="Vincent Rubinetti"
```

Comment:
Do you want to put my name in here too? Can lift things off the lab website if you'd like.